src/main/java/io/github/opencubicchunks/cubicchunks/world/level/levelgen/heightmap/surfacetrackertree/SurfaceTrackerNode.java

```diff
@@ -176,6 +131,15 @@ protected void setDirty(int idx) {
         dirtyPositions[idx >> 6] |= 1L << idx;
     }
 
+    public void setAllDirty() {
+        assert dirtyPositions.length == 4;
+
+        dirtyPositions[0] = -1;
+        dirtyPositions[1] = -1;
+        dirtyPositions[2] = -1;
+        dirtyPositions[3] = -1;
```

Comment:
Why didn't we use Arrays.fill here?

Reply:
No reason other than the possible tiny performance improvement